test/dynamo/test_export.py

```diff
@@ -2891,10 +2891,10 @@ def f(x):
 
         example_inputs = (torch.rand(5),)
         with self.assertRaisesRegex(
-            torch._dynamo.exc.UserError,
-            "Expected 4 arguments",
+            TypeError,
+            r"cond\(\) missing 1 required positional argument: 'operands'",
```

Comment:
This is a bit suspicious, why did the error message change?

Reply:
Oh, does it just go through regular Python and that's what raises the TypeError?

Reply:
yes, it hits the error at the python wrapper